OmiseSDK/Compatibility/OMSConstants.h

```diff
@@ -17,6 +18,7 @@ extern OMSSourceTypeValue const _Nonnull OMSSourceTypeValueInstallmentKTC;
 extern OMSSourceTypeValue const _Nonnull OMSSourceTypeValueInstallmentKBank;
 extern OMSSourceTypeValue const _Nonnull OMSSourceTypeValueInstallmentSCB;
 extern OMSSourceTypeValue const _Nonnull OMSSourceTypeValueEContext;
+extern OMSSourceTypeValue const _Nonnull OMSSourceTypeValueMobileBankingSCB;
```

Comment:
I think this is duplicated declaration from above.

Reply:
Removed 🙇